app/views/layouts/application.html.erb

```diff
@@ -40,9 +40,14 @@
         <ul class="nhsuk-header__navigation-list app-header__navigation-list">
           <% if current_user.present? %>
             <li class="nhsuk-header__navigation-item">
-              <div class="nhsuk-header__navigation-link app-u-text-decoration-none">
+              <a class="nhsuk-header__navigation-link" href="<%= sessions_path %>">
+                School sessions
+              </a>
```

Comment:
You're so analogue Paul! 😆 

```suggestion
              <%= link_to "School sessions", sessions_path, class: "nhsuk-header__navigation-link" %>
```

There's also a `govuk_link_to`.

But this isn't actually important, just a non-blocking suggestion.

Reply:
Ah, of course! Fixed.